examples/src/main/java/com/mapbox/navigation/examples/core/MapboxNavigationActivity.kt

```diff
@@ -431,22 +434,28 @@ class MapboxNavigationActivity : AppCompatActivity() {
         mapboxNavigation.startTripSession()
     }
 
+    @OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
     override fun onStart() {
         super.onStart()
         mapboxNavigation.registerRoutesObserver(routesObserver)
         mapboxNavigation.registerNavigationSessionStateObserver(navigationSessionStateObserver)
         mapboxNavigation.registerRouteProgressObserver(routeProgressObserver)
         mapboxNavigation.registerLocationObserver(locationObserver)
         mapboxNavigation.registerVoiceInstructionsObserver(voiceInstructionsObserver)
+        mapboxNavigation.setAdasisMessageCallback(AdasisConfig.Builder().build()) { message ->
```

Comment:
Hmm, wondering if this callback should only work when the app is in foreground.

Reply:
not really. NIO is going to build their ISA (intellectual speed assistance) based on this message flow. So it's crucial for them to be getting messages all the time while ACC is ON.

Reply:
Then we should adjust the example. But I'm not sure if we need it here at all. In the end we'd want a separate activity I guess.